solc/CMakeLists.txt

```diff
@@ -4,8 +4,21 @@ set(
 	main.cpp
 )
 
+if(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+	# Since we do not want to depend on networking code on production binaries,
+	# this feature is only available when creating a debug build.
+	# the LSP's TCP listener is exclusively used more convenient debugging.
+	option(SOLC_LSP_TCP "Solidity LSP: Enables TCP listener support (should only be eanbled for debugging purposes)." OFF)
+	if(SOLC_LSP_TCP)
+		set(SOLC_DEFINITIONS SOLC_LSP_TCP=1)
+		list(APPEND sources LSPTCPTransport.cpp LSPTCPTransport.h)
+		set(SOLC_LSP_LIBS pthread)
```

Comment:
I think it is not portable to directly use `pthread` directly here. 
You could do something like
```cmake
find_package(Threads REQUIRED)
set(SOLC_LSP_LIBS "${CMAKE_THREAD_LIBS_INIT}")
```

Reply:
you are right. thanks!